src/mon/MonCommands.h

```diff
@@ -419,7 +419,7 @@ COMMAND("mon dump " \
 	"name=epoch,type=CephInt,range=0,req=false", \
 	"dump formatted monmap (optionally from epoch)", \
 	"mon", "r", "cli,rest")
-COMMAND("mon stat", "summarize monitor status", "mon", "r", "cli,rest")
+COMMAND("mon status", "summarize monitor status", "mon", "r", "cli,rest")
```

Comment:
commands should not simply be removed. Before removing a command, it needs to be marked as DEPRECATED. If the consensus is that we should get rid of `mon stat`, please DEPRECATE it first.